apps/explorer/src/ui/Card.tsx

```diff
@@ -10,10 +10,12 @@ const cardStyles = cva(null, {
             default: 'bg-gray-40',
             highlight: 'bg-success-light',
             lightBlue: 'bg-sui/10',
+            white: 'bg-white border border-steel',
```

Comment:
I feel like I've seen other in-flight PRs add this but with different styles.

Reply:
I just took around to other PRs currently, saw @williamrobertson13 added some styling on `Card` but not to the bg color. I will merge this so others can rebase if there are pending changes related.